### ISSUE TITLE
False positive in `private_unit_test` rule

### ISSUE THREAD

**Issue by blackm00n** [realm/SwiftLint#1532]
When test method has parameters, e.g.:
```
private testSmth(_ smth: Smth, file: StaticString = #file, line: UInt = #line)
```
there should not be no warning, as intention that it's a utility method is obvious, and it cannot cause "test not run" problem.
